AsyncDisplayKit/ASCollectionNode.h

```diff
@@ -33,19 +33,19 @@ NS_ASSUME_NONNULL_BEGIN
  *
  * @discussion Initializes and returns a newly allocated collection node object with the specified layout.
  *
- * @param layout The layout object to use for organizing items. The collection view stores a strong reference to the specified object. Must not be nil.
+ * @param layout The layout object to use for organizing items. The collection node stores a strong reference to the specified object. Must not be nil.
  */
 - (instancetype)initWithCollectionViewLayout:(UICollectionViewLayout *)layout;
 
 /**
  * Initializes an ASCollectionNode
  *
- * @discussion Initializes and returns a newly allocated collection node object with the specified frame and layout.
+ * @discussion Initializes and returns a newly allocated collection node object with the specified layout.
  *
- * @param frame The frame rectangle for the collection view, measured in points. The origin of the frame is relative to the superview in which you plan to add it. This frame is passed to the superclass during initialization.
- * @param layout The layout object to use for organizing items. The collection view stores a strong reference to the specified object. Must not be nil.
+ * @param layout The layout object to use for organizing items. The collection node stores a strong reference to the specified object. Must not be nil.
+ * @param layoutInspector The layout inspector to use, or nil to use the ASDK default layout inspector.
  */
-- (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionViewLayout *)layout;
+- (instancetype)initWithCollectionViewLayout:(UICollectionViewLayout *)layout layoutInspector:(nullable id<ASCollectionViewLayoutInspecting>)layoutInspector;
```

Comment:
@Adlai-Holler this may sound silly, but what do you think about including an initializer right below this, without another block comment, that is -initWithCollectionViewFlowLayout: ?  I think that would be more clear for folks who use the common case of a flow layout and don't know what this layoutInspector thing is -- and flow layouts most definitely aren't going away, at least not before the layoutInspector is :)